client/www/pages/docs/strong-init.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Strong Init (experimental)
+---
+
+What if you could use the types you defined in `instant.schema.ts`, inside `init`? We have a new version of `init` out that lets you do this. It's in beta, but if you use it, `useQuery` and `transact` will automatically get typesafety and intellisense.
+
+Here's how it works
+
+If you want to migrate, swap out `init` with `init_experimental` and pass it your app's graph schema object.
+
+```ts
+import { init_experimental } from '@instantdb/react';
+// 1. Import your schema file
+// Don't have a schema? check out https://www.instantdb.com/docs/cli to get started
+import schema from '../instant.schema.ts';
+
+const db = init_experimental({
+  appId: '__APP_ID__',
+  // 2. Use it inside `init_experimental`
+  schema,
+});
+
+export default function App() {
+  // 3. now useQuery is strongly typed!
+  const todosRes = db.useQuery({
+    todos: {},
+  });
+
+  function addTodo({ title: string }) {
+    db.transact([
+      // 4. mutations are typechecked too!
+      db.tx[id()].update({ title }),
+    ]);
+  }
+
+  return <TodoApp todosRes={todosRes} onAddTodo={addTodo} />;
+}
+```
+
+## Changes to `tx`
+
+If you used Instant before, you would use the global `tx` object. Instead, use the schema-aware `db.tx`.
+
+```ts
+// ❌ instead of global `tx`
+tx.todos[id()].update({ done: true });
+
```

Comment:
Nit. Let's remove this, I remember when I tried to do a similar think w/ the merge docs (saying don't do this, do this instead) users found it confusing in the docs

```suggestion
```